docs/v1/user-story

```diff
@@ -0,0 +1,123 @@
+systems or network errors.  But, I did see that latency for the message PUT API calls was through the roof.  I went to see 
+what was wrong in my app and why it was so slow, and with a bit of looking at logs, I found that I was idling in synchronous 
+transactions to the database.  I didn’t want that in the critical path for message delivery, but again, the support team did 
+not want anything to happen without a log of it.  But, at least I know where the problem is.  I probably saved hours of 
+poking around the system with that service graph.  Now to find out what the problem is with the database vendor.
+
+## 10. Update deployment configuration
+I called the database vendor support team and they quickly told me the problem.  Utilization is going through the roof.  I 
+didn’t notice on the node.js app itself because kubernetes was auto-scaling that for me without a hitch.  (Well, lots of 
+instances now, but, hey, that’s between finance and the cloud - I just need to keep support tickets down.)  But, the 
+database was not scaling because we only bought the “small” configuration.  I quickly got the permission to purchase the 
+upgrade, and issued the single command to update service instance with the “large” argument, and after an hour of the update 
+automatically sharding and moving data around, everything was better.  It’s really nice how all that was taken care of for 
+me.  Some database vendor employee must have worked really hard on that “update” service broker.  I’m glad I did not have to 
+solve that.
+
+## 11. Discover type update  
+Just when I thought everything was OK, I get a message from kubernetes that my app is out of date.  How did that happen?  
+Well, it turns out that the database vendor rolled out a new version of the database due to a critical security bug fix.  If 
+this was a fully managed service like SFDC, they would just fix it quietly and not have to notify me, but I insisted on 
+using a service that lets me have my own instance and know that it’s completely isolated from other customers, so when there 
+is an update, I can choose whether or not to deploy it.  I’m now starting to think harder about the benefits of transparency 
+in vendor deployments vs. the benefits of fully managed services.  But I have to admit just how cool it is that I got the 
+message so quickly in such an obvious way.  They made an update and published it to the store.  Kubernetes’ record of the 
+database type in the service catalog that was created when I bought the service noticed the new store entry.  I was notified 
+me about the update immediately since I had an instance of it running and my chat app bound to that instance.  Now to do the 
+DB upgrade... 
+
+## 12. Upgrade service instances to new service type version
+Wow, the version upgrade was just as easy as the “small” to “large”!   All I needed was one update command and the database 
```

Comment:
Since the service is managed behind the broker, the narrator has no idea.  I can imagine what a DB upgrade would look like and the narrator probably can as well.  But, the complexity is hidden behind the broker, which is really the whole point and the power of the model.  

To your point, though, the reason that I have a story around the narrator's upgrade of his own code is because, as you note, we need to see that as readers.  Can you defer your interest in this to that later point in the story?